packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -31,22 +24,13 @@ describe('<Slider />', () => {
     return;
   }
 
-  const mount = createMount();
   let classes;
   const render = createClientRender();
 
   before(() => {
-    classes = getClasses(<Slider value={0} />);
+    classes = {};
   });
 
-  describeConformance(<Slider value={0} />, () => ({
```

Comment:
The conformance tests are moved to the `SliderUnstyled.test.js`

Reply:
Each component should run this test suite. If a component is not running it then it doesn't share common functionality with the other components tested with this suite.

Reply:
Added